creator-node/src/routes/tracks.js

```diff
@@ -246,6 +274,37 @@ module.exports = function (app) {
 
       // if track created, ensure files exist with trackuuid = null and update them.
       if (trackCreated) {
+        // Update the transcoded 320kbps copy
+        if (transcodedTrackUUID) {
```

Comment:
error in else case?

Reply:
in order to be backwards compat with older version of the dapp, we shouldn't error in that case. we can roll that change in after this goes out though

Reply:
ah good point